docs/src/pages/docs/how-it-works.md

```diff
@@ -7,7 +7,7 @@ layout: ../../layouts/DocsLayout.astro
 
 ## Simple explanation
 
-`whyframe` extracts out the HTML within `iframe`s, which is processed as a separate module by the bundler, and the `iframe` would then load a different HTML with a custom script that loads that separate module as a new entrypoint.
+`whyframe` extracts out the HTML within `iframe`, which is processed as a separate module by the bundler, and the `iframe` would then load a different HTML with a custom script that loads that separate module as a new entrypoint.
```

Comment:
```suggestion
`whyframe` extracts the HTML within an `iframe`, which is processed as a separate module by the bundler, and the `iframe` would then load a different HTML with a custom script that loads that separate module as a new entrypoint.
```

I think we need an `an` here to connect it, but making it singular makes sense as the next sentence is also singular 👍